### ISSUE TITLE
Dilated bowel

### ISSUE THREAD

**Issue by sng2140** [obophenotype/human-phenotype-ontology#7735]
**Preferred term label:**
Dilated bowel

**Synonyms**

**Definition (free text, please give PubMed ID)**
Fetal bowel dilatation is characterized by fluid-filled intestinal loops which measure at least 15 mm in length or 7 mm in diameter (PMID: 26819789) Dilated small bowel, dilated large bowel, dilated bowel

**Parent term (use hpo.jax.org/app)**
abnormal fetal gastrointestinal system morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Omim: 619351 MEGACYSTIS-MICROCOLON-INTESTINAL HYPOPERISTALSIS SYNDROME 2; MMIHS2
 

**Your nano-attribution (ORCID)**

**Comment by sng2140** [comment]
Meconium ileus can be a differential to this, prenatally

**Comment by pnrobinson** [comment]
@sng2140

Added as HP:4000140

Definition:Fetal bowel dilatation is characterized by fluid-filled intestinal loops which measure at least 15 mm in length or 7 mm in diameter. Ultrasonographic image of dilated fetal bowel is a sign of intestinal mechanical or functional obstruction and its prevalence will depend on the underlying condition: bowel atresia or stenosis, malrotation with volvulus, meconium ileus, total colonic aganglionosis, and meconium plug syndrome. The etiology of fetal bowel dilatation is a difficult prenatal diagnosis since ultrasound has limited accuracy for bowel evaluation.
